_pages/install/integrations/opf-integration/opf-child-pages/2-quickbuy.md

```diff
@@ -0,0 +1,134 @@
+```
+
+### Adding CMS Component Manually
+
+This section describes how to add the quick buy CMS component to Spartacus using ImpEx.
+
+You can enable quick buy by adding the Quick Buy button component to the Cart page/You can enable the checkout components for open payment framework with the following ImpEx:
+
+```text
+INSERT_UPDATE CMSFlexComponent;$contentCV[unique=true];uid[unique=true];name;flexType
+;;OpfQuickBuyButtonsComponent;Opf Quick Buy Buttons Component;OpfQuickBuyButtonsComponent
+
+INSERT_UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];name;cmsComponents(uid, $contentCV)
+;;CenterRightContentSlot-cartPage;Center Right Content Slot for Cart Page;CartTotalsComponent,CartApplyCouponComponent,CartQuickOrderFormComponent,OpfQuickBuyButtonsComponent,CartProceedToCheckoutComponent
+```
+
+## Configuring Quick Buy
+
+### Configuring Quick Buy for Apple Pay
+
+The open payment framework enables you to implement an express checkout for your customers using Apple Pay.
+
+#### Browser Support
+
+Quick buy for Apple Pay is available on Safari. For more information, see https://developer.apple.com/documentation/apple_pay_on_the_web.
+
+#### Modifying the Quick Buy Button
+
+You can modify the appearance of the Apple Pay quick buy button. By default, the quick buy button is black with a 'Buy with Apple Pay' label. The default values of the Apple Pay button style are as follows:
+
+-`webkit-appearance`: `apple-pay-button`
+-`apple-pay-button-type`: `buy`
+-`apple-pay-button-style`: `black`
+
+To modify the Apple Pay button, you extend the style with the `%cx-opf-apple-pay` placeholder selector and overwrite the `apple-pay-button` class with the desired values.
+
+For the list of attributes and possible values for the Apple Pay quick buy button, see https://developer.apple.com/documentation/apple_pay_on_the_web/displaying_apple_pay_buttons_using_css.
+
+For more information on creating CSS placeholder selectors, see
+https://help.sap.com/docs/SAP_COMMERCE_COMPOSABLE_STOREFRONT/eaef8c61b6d9477daf75bff9ac1b7eb4/a95f88362b1a4ecf998047c93ab7bc12.html#loio23416cbcc1da4c9eaf513999dde8b7de.
+
+#### Modifying Card Parameters
+
+By default, the card parameters configuration is hard coded with `ApplePayService`. You can overwrite it by extending `ApplePayService` and overwriting the following object:
+
+```ts
+  protected readonly defaultApplePayCardParameters: any = {
+    shippingMethods: [],
+    merchantCapabilities: ['supports3DS'],
+    supportedNetworks: ['visa', 'masterCard', 'amex', 'discover'],
+    requiredShippingContactFields: ['email', 'name', 'postalAddress'],
+    requiredBillingContactFields: ['email', 'name', 'postalAddress'],
+  };
+```
+
+For more information on extending services in Spartacus, see https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/aa417173fe4a4ba5a473c93eb730a417/465a25442fd64a1ab33d98362d66d25b.html?q=extend%2520service.
+
```

Comment:
@FollowTheFlo @Matejk00 This is a link to documentation for SAP Commerce Cloud -- I could be wrong but I don't think this would apply to the storefront. Can you double check with @Platonn to see if he knows about any documentation we might have for extending a service in the frontend. I myself am not aware that we have anything like that 🤷 🙂

Reply:
True, this link is not valid. Instead we can put a reference to this SPA doc section: https://help.sap.com/docs/SAP_COMMERCE_COMPOSABLE_STOREFRONT/eaef8c61b6d9477daf75bff9ac1b7eb4/864a3158bf9f49c99e6196e4e0d27323.html?locale=en-US&q=extend+composable#customizing-services

It explains exactly what extensibility possibilities we have in SPA.

Reply:
Perfect! Thank you so much!!